Chapter_6/Readme.md

```diff
@@ -0,0 +1,29 @@
+###  http://ruby.learncodethehardway.org/book/ex6.html
+Study Drills:
+1) Add your comments to ex6.rb
+2) List these places in Readme.md
+3) Was he lying?
+4) Add this explanation to Readme.md
+5) Answer this in the Readme.md
+
+Go through this program and write a comment above each line explaining it.
+
+Find all the places where a string is put inside a string. There are four places.
```

Comment:
Please list the instances where you see this...or line number, whichever you prefer